app/controllers/shift_commerce/static_pages_controller.rb

```diff
@@ -0,0 +1,37 @@
+module ShiftCommerce
+  class StaticPagesController < ShiftCommerce::ApplicationController
+
+    include ShiftCommerce::TemplateDefinitionHelper
+    cache_shared_page only: :show
+
+    API_STATIC_PAGE_INCLUDES = "template_definition,meta.*".freeze
+
+    def show
+      fetch_static_page
+      set_static_page_meta_tags
+      render_static_page
```

Comment:
This action has been split into small, discrete functions to make it easier for developers to override individual pieces of functionality without having to override things they don't need to change.